### ISSUE TITLE
npm ERR! phantomjs-prebuilt@2.1.3

### ISSUE THREAD

**Issue by finid** [linnovate/mean#1481]
This an error I've come across when trying to install a MEAN stack on CentOS 7, and it occurs whether I'm using `mean-cli` or the standard MEAN boilerplate from meanjs.org.
The error is:

```
testing binary.
Binary is fine; exiting.
npm ERR! Linux 3.10.0-229.20.1.el7.x86_64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE

npm ERR! phantomjs-prebuilt@2.1.3 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the phantomjs-prebuilt@2.1.3 install script 'node install.js'.
npm ERR! This is most likely a problem with the phantomjs-prebuilt package,
npm ERR! not with npm itself.
```

It also happens even when I use node 5.5 and npm 3.x branch, so the solution given for [issue #433](https://github.com/linnovate/mean/issues/433) does not apply.

**Comment by sethbergman** [comment]
I had an issue with the boilerplate template using the mean-cli a couple weeks ago. I had to pull in the new dependency versions in package.json from upstream, but it wouldn't allow me to just use `git fetch upstream`, or `git pull`. Here's what worked for me.

I added a new remote named `old`, then pulled in the new package versions from upstream.
```
git remote add old https://github.com/linnovate/mean.git
git fetch old
```
Then I committed the new dependency versions to my repository, and this worked for me. Hope this helps.
